lib/spree_avatax_certified/engine.rb

```diff
@@ -29,10 +33,33 @@ class Engine < Rails::Engine
       end
     end
 
+    def self.configure_caching
+      SpreeAvataxCertified.cache = if ENV['MEMCACHED_SERVERS'].blank?
```

Comment:
@glongman actually instead of an env var you have to load it from config/memcached.yml

```ruby
# parse the memcached.yml
 memcached_config = YAML.load_file(Rails.root.join('config/memcached.yml'))
 memcached_hosts = memcached_config['defaults']['servers']
```